packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -1,5 +1,4 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import type { Prisma } from "@prisma/client";
 import type { calendar_v3 } from "googleapis";
```

Comment:
This has been replaced also, check with @hbjORbj - should be more performant henseforth.